training/src/anemoi/training/train/train.py

```diff
@@ -207,7 +207,7 @@ def model(self) -> pl.LightningModule:
             "truncation_data": self.truncation_data,
             "metadata": self.metadata,
             "statistics": self.datamodule.statistics,
-            "statistics_tendencies": self.datamodule.statistics_tendencies,
+            "statistics_tendencies": self.datamodule.statistics_tendencies,  # should depend on config
```

Comment:
What do you mean with `# should depend on config`?

Reply:
I bet we are passing them because of the tendencies scalers, guess in the future we could construct that kwargs settings based on the config. (so if the config has the Tendency Scalar then we add them). Ultimately the `create_scalars` function accept  `**kwargs` too

@HCookie